presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -163,6 +164,18 @@ public static PlanMatchPattern aggregation(
         return result;
     }
 
+    public static PlanMatchPattern markDistinct(
+            String markerSymbol,
+            List<String> distinctSymbols,
+            Optional<String> hashSymbol,
+            PlanMatchPattern source)
+    {
```

Comment:
What do you think about having two methods, one with hashSymbol and one without? That way you would need to pass Optional (at least into the API of `PlanMatchPattern`).